### PR TITLE
Add warnings info and missing notification

### PR DESCRIPTION
**Pull request by Raluca-Cruceru**
@jgrosseo

### CODE REVIEW

File: docs/hyperloop/userdocumentation.md [hunk 1]
```diff
@@ -269,7 +269,81 @@ When creating or enabling wagons, you can use a pull request instead of a packag
...
+ 
+ * The maximum PSS memory consumption is larger than 30% of the average PSS, therefore the train cannot be automatically composed. The test will be checked by the operator and, if there is no memory leak, the train can be composed. Otherwise, they will advise the user to check for possible causes and improvements before requesting again.
 
+ ### 3. <a name="warning-cpu"></a> CPU usage too large
+ 
+  <div align="center">
+    <img src="../images/warningCPU.png" width="40%">
+  </div>
+ 
+ * The CPU usage limit is set per dataset and all trains running on a specific dataset must respect this constraint. If the limit is not respected, the train cannot be composed without PWG approval. Therefore, the user should discuss the details and requirements for this train with the PWG before requesting again.
```
Comment (jgrosseo) [inline docs/hyperloop/userdocumentation.md:304]:
```suggestion
 * The CPU usage limit is set per dataset and all trains running on a specific dataset must respect this constraint. If the limit is not respected, the train cannot be composed without PWG approval. Therefore, the user should discuss the details and requirements for this train with the PWG before requesting again. Depending on the amount of total resources, an approval in the Physics Board (PB) may also be needed.
```

File: docs/hyperloop/userdocumentation.md [hunk 2]
```diff
@@ -269,7 +269,81 @@ When creating or enabling wagons, you can use a pull request instead of a packag
...
+ * The log file is too large, therefore the train cannot be composed, and the user should check for factors leading to this.
+ 
+ ### 7. <a name="warning-derived-output"></a> Derived output too large for slim train
+ 
+  <div align="center">
+    <img src="../images/warningDerivedOutput.png" width="40%">
+  </div>
+ 
+ * This is specific to tests with wagons set as ready for slim derived data. It can happen when the estimate is slightly different during the wagon test and the train test. If the output is lower than 5000 MB, the operator can submit the train on request. 
+ * Otherwise, the user is advised to switch to standard derived data by unchecking the option “Ready for slim derived data” in the wagon edit view. Then a request for standard derived data train can be made.
```
Comment (jgrosseo) [inline docs/hyperloop/userdocumentation.md:337]:
```suggestion
 * This is specific to tests with wagons set as ready for slim derived data. As the entire output is merged into one single file, there is a limit of 4000 MB for. The user is advised to switch to standard derived data by unchecking the option “Ready for slim derived data” in the wagon edit view. Then a request for standard derived data train can be made.
```
(I think we should focus here on the user view, not what the operator would do if it appears in the train test)

File: docs/hyperloop/userdocumentation.md [hunk 3]
```diff
@@ -269,7 +269,81 @@ When creating or enabling wagons, you can use a pull request instead of a packag
...
+ 
+## <a name="warnings"></a> Warnings
+ 
+When a wagon test finishes in warning, this means that the wagon will not be included in the automatic composition schedule. Therefore, train composition can be requested in the Operation channel, where an operator will take care of the request. Depending on the nature of the warning and the degree of exceeding specific constraints, the operator will either compose your train or advise you to review and improve certain parts before requesting a train again. In the latter case, the user can analyze the test and review the logs, searching for ways of improving resource usage or other elements that caused the exceptions.
```
Comment (jgrosseo) [inline docs/hyperloop/userdocumentation.md:275]:
```suggestion
When a wagon test finishes in warning, this means that the wagon will not be included in the automatic composition schedule. Therefore, train composition can be requested in the Operation channel, where an operator will take care of the request. Before doing so, please review if you cannot fix the cause of the warning yourself. Depending on the nature of the warning and the degree of exceeding specific constraints, the operator will either compose your train or advise you to review and improve certain parts before requesting a train again. In the latter case, the user can analyze the test and review the logs, searching for ways of improving resource usage or other elements that caused the exceptions.
```

**Review by jgrosseo — COMMENTED** [review]
Thanks! I made few suggestions. You can simply accept them on github and then we can merge

**Comment by Raluca-Cruceru** [comment]
> Thanks! I made few suggestions. You can simply accept them on github and then we can merge

Hi, thanks for the suggestions, I accepted them. :)

**Pull request merged**